.github/workflows/ceph-s3-tests.yaml

```diff
@@ -0,0 +1,80 @@
+name: ceph-s3-tests
+on: 
+  push:
+    branches:
+      - master
+    paths:
+      - 'src/*'
+  pull_request:
+      branches:
+      - master
+      paths:
+      - 'src/*'
```

Comment:
I don't think we should trigger this on every push and pull request.
maybe nightly.
@dannyzaken @nimrod-becker

Reply:
In order to collect observation without interrupting the team's work, this action would run manually and scheduled (twice during the night).